src/components/Comments/ReviewComments.jsx

```diff
@@ -0,0 +1,39 @@
+import { useState, useEffect } from 'react'
+import { useParams } from 'react-router-dom'
+import { getCommentsOnReview } from '../../utils/api'
+
+const ReviewComments = () => {
+  const [reviewComments, setReviewComments] = useState([])
+  const [isLoading, setIsLoading] = useState(true)
+  const { review_id } = useParams()
+
+  useEffect(() => {
+    setIsLoading(true)
+    getCommentsOnReview(review_id).then((reviewCommentData) => {
+      setReviewComments(reviewCommentData)
+      setIsLoading(false)
+    })
+  }, [review_id])
+
+  return isLoading ? (
+    <h2>ReviewComments: Loading Animation Goes Here</h2>
+  ) : (
+    <section className="comment-container">
```

Comment:
Currently it is not very clear what is the comments and what is the review contents - I would add something to specify this is the comments section or change the css so it is broken up - along with the users name who has left the comment and date/time etc. 

I also would like there to be a default message if there are no comments on that review yet.